webui/react/src/components/ResourcePoolCard.tsx

```diff
@@ -153,6 +153,7 @@ const ResourcePoolCard: React.FC<Props> = ({ resourcePool: pool }: Props) => {
           rpBindingFlagOn && canManageResourcePoolBindings
             ? [
                 {
+                  disabled: pool.defaultAuxPool || pool.defaultComputePool,
```

Comment:
When button is disabled, click event is bubbling to the card

https://github.com/determined-ai/determined/assets/109113616/23b213b7-d2df-49bb-89e5-a89c2af2ab4d